src/interface/graphql/modules/user/user.access-control.ts

```diff
@@ -51,6 +51,14 @@ export class UserAccessControl extends AccessControl {
       })
   }
 
+  public async publicIsTeamLeader(user: UserWithContext): Promise<boolean> {
```

Comment:
Eu sei que eu que sugeri o public... kk mas vendo agora nesse contexto, acho que isUserTeamLeader faz mais sentido.

```suggestion
  public async isUserTeamLeader(user: UserWithContext): Promise<boolean> {
```